.github/PULL_REQUEST_TEMPLATE.md

```
@@ -0,0 +1,40 @@
+<!--
+  文書を変更した場合、以下を追加すること
+  - [ ] `docs/documentation-management.md`の手動確認を実施した
+-->
+
+## Related Issue / Links
+
+<!-- 関連するIssueを列挙する -->
+
+- closed #
+
+- []()
```

markdownlint-cli2 (0.23.2)

[warning] No empty links

(MD042, no-empty-links)